kbpagesconfig/editor.go

```diff
@@ -0,0 +1,206 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/keybase/client/go/minterm"
+	"github.com/keybase/kbfs/libpages/config"
+	"github.com/sergi/go-diff/diffmatchpatch"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const configFileName = ".kbp_config"
+
+var term *minterm.MinTerm
+
+func init() {
+	var err error
+	if term, err = minterm.New(); err != nil {
+		fmt.Fprintf(os.Stderr, "opening terminal error: %s\n", err)
+		os.Exit(1)
```

Comment:
It would be nice to have some tests for this editor stuff, but this might prevent it?  Any way around it?  We can probably pass the term in from elsewhere, maybe as an interface, and a test could provide one that always says yes or no.  Then `minterm.New()` would be done by the CLI code that calls into here.